src/main/resources/resource/Arduino/MrlComm/MrlNeopixel.h

```diff
@@ -1,45 +1,11 @@
 #ifndef MrlNeopixel_h
 #define MrlNeopixel_h
 
+#include "Adafruit_NeoPixel.h"
 #include <Arduino.h>
 
-/***********************************************************************
- * NEOPIXEL DEFINE
- */
-//timing for the neopixel communication
-#define T1H 900    
-#define T1L 600
-#define T0H 400
-#define T0L 900
-#define RES 6000
-
-#define NS_PER_SEC (1000000000L)
-#define CYCLES_PER_SEC (F_CPU)
-#define NS_PER_CYCLE ( NS_PER_SEC / CYCLES_PER_SEC )
-#define NS_TO_CYCLES(n) ( (n) / NS_PER_CYCLE )
-#ifndef ESP8266
-
-// Arduino Mega Pins
-#if defined(ARDUINO_AVR_MEGA2560) || defined(ARDUINO_AVR_ADK)
-  #define digitalPinToSendBit(P,V) \
-    (((P) >= 22 && (P) <= 29) ? sendBitA(V) : \
-        ((((P) >= 10 && (P) <= 13) || ((P) >= 50 && (P) <= 53)) ? sendBitB(V) : \
-        (((P) >= 30 && (P) <= 37) ? sendBitC(V) : \
-        ((((P) >= 18 && (P) <= 21) || (P) == 38) ? sendBitD(V) : \
-        ((((P) >= 0 && (P) <= 3) || (P) == 5) ? sendBitE(V) : \
-        (((P) >= 54 && (P) <= 61) ? sendBitF(V) : \
-        ((((P) >= 39 && (P) <= 41) || (P) == 4) ? sendBitG(V) : \
-        ((((P) >= 6 && (P) <= 9) || (P) == 16 || (P) == 17) ? sendBitH(V) : \
-        (((P) == 14 || (P) == 15) ? sendBitJ(V) : \
-        (((P) >= 62 && (P) <= 69) ? sendBitK(V) : sendBitL(V)))))))))))
-
-#else
-  #define digitalPinToSendBit(P,V) \
-    (((P) >= 0 && (P) <= 7) ? sendBitD(V) : (((P) >= 8 && (P) <= 13) ? sendBitB(V) : sendBitC(V)))
-#endif
-#else
-  #define digitalPinToSendBit(P,V) return
-#endif
+// TODO: Why two enums to tell the animation to stop?  maybe best to just
```

Comment:
we should remove animation 0 and 1.. they aren't animations..they are control messages.. and if you want to stop an animation you can say "clear()"   if we want an actual stopAnimiation() call, we should add that to just stop the current animation.

Reply:
implemented animation 1 as it previously existed as control message - stop animation ... clear still stops and clears ... 0 is meaningless as it was before